packages/contracts/src/dollar/core/UbiquityDollarManager.sol

```diff
@@ -21,10 +21,8 @@ import "./TWAPOracleDollar3pool.sol";
 contract UbiquityDollarManager is AccessControl {
     using SafeERC20 for IERC20;
 
-    bytes32 public constant GOVERNANCE_TOKEN_MINTER_ROLE =
-        keccak256("GOVERNANCE_TOKEN_MINTER_ROLE");
-    bytes32 public constant GOVERNANCE_TOKEN_BURNER_ROLE =
-        keccak256("GOVERNANCE_TOKEN_BURNER_ROLE");
+    bytes32 public constant UBQ_MINTER_ROLE = keccak256("UBQ_MINTER_ROLE");
+    bytes32 public constant UBQ_BURNER_ROLE = keccak256("UBQ_BURNER_ROLE");
```

Comment:
Looks like the role names were reverted to pre-updated-renames.

Reply:
fixed